best403unlocker

```diff
@@ -8,11 +8,49 @@ if [ "$EUID" -ne 0 ]; then
   exit 1
 fi
 }
+install_required_packages(){
+    if [[ -f /etc/os-release  ]]; then
+        source /etc/os-release
+
+        case $ID in
+              debian|ubuntu|mint|elementary)
+                apt install -y wget
```

Comment:
in debian base OSes you have to use ```apt update -y``` first then you can install your desired packages

Reply:
Done.